.github/workflows/dotnet-ci.yml

```diff
@@ -30,6 +30,14 @@ jobs:
           dotnet-version: |
             8.0.x
             9.0.x
+      - name: Set up Python
```

Comment:
We should add these steps to `dotnet-main.yml` and `dotnet-release.yml`

Reply:
Yes. I've only added them to dotnet-ci to make sure the steps were correct. Now that I have tested the ci is correct I'll replicate on main and release